infra/k8s/backup.yaml

```diff
@@ -26,11 +26,13 @@ spec:
         spec:
           containers:
           - name: backup
-            image: gcr.io/clusterfuzz-images/base
+            image: gcr.io/clusterfuzz-images/base:vguidi_py311_testing_v1
```

Comment:
this is intentional right?

Reply:
Yes. If I changed latest, everybody else that is not on 311 right now would break